DataFormats/StdDictionaries/src/classes_def_pair.xml

```diff
@@ -40,6 +40,7 @@
  <class name="std::pair<short,short>"/>
  <class name="std::pair<short,std::vector<short> >"/>
  <class name="std::pair<short,unsigned int>"/>
+ <class name="std::pair<short,int>"/>
```

Comment:
Just to make sure: here I use `short`, but in #47030 I used `int16_t`. Do I leave things as they are, or is it better to be consistent (if so, would you suggest to change type here, or in [1]) ?

[1] https://github.com/cms-sw/cmssw/blob/bf4d4fe891e7d94189bb86054451336cfba119c9/DataFormats/L1TGlobal/interface/GlobalObjectMapFwd.h#L28

Reply:
I'm of the opinion that you should keep it the way it is. The DataFormats/StdDictionaries is meant to allow sharing of the dictionaries across different packages. If another package has the same use of `std::pair` they might only declare it as `short`. Given `short` is the actual fundamental type, I'd stick with that. The use of `int16_t` in the typedef is also useful since it is effective documentation stating why they want that size.